docs/specification/02_event_service.md

```
@@ -223,6 +223,27 @@ Reservation Service가 호출합니다.
 }
 ```
 
+### 2.2 대기열 정리 대상 회차 조회
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)